### PR TITLE
Laura HTML file 

### PR DESCRIPTION
**Pull request by lqueipom**
Modifying HTML file to better fit my portfolio's description (will add more in depth information later)

### CODE REVIEW

**Comment by LukeGMTravis** [comment]
@Tetheta Starting review

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,22 +2,32 @@
...
-      <h1>Stanley 🐈</h1>
-      <p>This is Stanley.</p>
-      <img src="images/stanley-1.jpg"/>
+      <h1>Laura's Portfolio</h1>
+      <p>This is Laura, I am a rising junior at MIT. I am currently majoring in Computer Science. 
+	I was born and raised in Venezuela and moved to the US to pursue higher education.</p>
+      <img src="images-me/IMG_0469.JPG"/>
+      <h2>Projects</h2>
+      <ul>
+	<li> Worked at the University of Catalonia on an Autism related project.
```
Comment (LukeGMTravis) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:16]: Linting: Remove leading space

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,22 +2,32 @@
...
+	<li>
```
Comment (LukeGMTravis) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:17]: This should be a closing tag (and should probably be on the previous line).

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html (hunk 1 above)
Comment (lqueipom) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:16]: Hi Luke! Thank you for the comments. What do you mean specifically by linting?

Comment (LukeGMTravis) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html]: Ah, linting is essentially checking for formatting issues. (Issues not necessarily meaning that  the code is bad, just that it isn't formatted in the standard way.) https://en.wikipedia.org/wiki/Lint_(software)
wtf/lint

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html [hunk 3]
```diff
@@ -13,8 +13,7 @@ <h1>Laura's Portfolio</h1>
       <img src="images-me/IMG_0469.JPG"/>
       <h2>Projects</h2>
       <ul>
-	<li> Worked at the University of Catalonia on an Autism related project.
-	<li>  
+	<li> Worked at the University of Catalonia on an Autism related project.</li>
```
Comment (LukeGMTravis) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:16]: Still some extra space after the <li>

Comment (lqueipom) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:16]: ohh I think I got it now, you mean that it should be <ul><li>text</li></ul>?

Comment (lqueipom) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:16]: ohhh I think I got it now, do you mean between the ul closing tag and the p opening tag? (it is my first HTML tag so I am still getting used to it :)

Comment (LukeGMTravis) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html]: Oh no no sorry! I didn't notice that github formatted my list tag into a bullet point haha. What you had before was fine, I was just meaning that the space before "Worked" should be removed

Comment (lqueipom) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html]: Ohh hahaha, got it! I think it should be okay now, let me know if there is anything else I should change.

**Pull request merged**